integration.go

```diff
@@ -19,6 +21,15 @@ type AWSIntegrationFragment struct {
 	OwnershipTagKeys     []string `graphql:"ownershipTagKeys"`
 }
 
+type AzureResourcesIntegrationFragment struct {
+	TenantId                                             string        `graphql:"tenantId"`
+	SubscriptionId                                       string        `graphql:"subscriptionId"`
+	LastSyncedAt                                         *iso8601.Time `graphql:"lastSyncedAt"`
+	Aliases                                              []string      `graphql:"aliases"`
+	AllowManualSyncInfrastructureResources               bool          `graphql:"allowManualSyncInfrastructureResources"`
+	MinutesUntilManualSyncInfrastructureResourcesAllowed *int          `graphql:"minutesUntilManualSyncInfrastructureResourcesAllowed"`
```

Comment:
are these actually going to be public graphql fields?

Reply:
Cannot really tell from the graphql schema file, so I am reaching out to confirm.

Reply:
everything but `allowManualSyncInfrastructureResources` and  `minutesUntilManualSyncInfrastructureResourcesAllowed` will be in the public schema

Reply:
ok cool - make sure we only add whats going to be in the public schema